test/services/teams/services/admin.test.js

```diff
@@ -72,33 +72,39 @@ describe('\'/teams/manage/admin\' service', () => {
 				total: 1,
 			};
 
-			expectedResult = [{
-				membersTotal: 1,
-				name: 'spinatenpower',
-				_id: teamId,
-				color: '#d32f2f',
-				desciption: 'Spintatenpower makes teams greate again',
-				createdAtMySchool: true,
-				hasMembersOfOtherSchools: false,
-				hasRocketChat: false,
-				createdAt,
-				ownerExist: true,
-				schools: [{
-					_id: sessionSchoolId,
-					name: 'Schiller-Oberschule',
-				}],
-				schoolMembers: [
-					{
-						role: 'teamowner',
-						user: {
-							_id: userId,
-							firstName: 'Hans',
-							lastName: 'Peter',
-							roles: ['lehrer'],
+			expectedResult = {
+				limit: 50,
+				skip: 0,
+				total: 1,
+				data:[{
```

Comment:
Codacy found an issue: [Missing space before value for key 'data'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5734878)